editor/src/clj/editor/lsp.clj

```diff
@@ -442,11 +450,49 @@
                 (lsp.async/with-auto-evaluation-context evaluation-context
                   (reduce #(sync-resource-state! %1 %2 evaluation-context) state polled-resources)))))
 
+(let [method (-> Globs
+                 (.getDeclaredMethod "toUnixRegexPattern" (into-array Class [String]))
+                 (doto (.setAccessible true)))]
+  (defn- ^Pattern make-glob-pattern [s]
```

Comment:
We usually put the return type between the function name and the argument list, but perhaps this works too?

Reply:
Yes, both work. I'll move the type hint to the arglist!